.cirrus/tasks.yml

```diff
@@ -48,6 +54,55 @@ clone_script_template: &CLONE_SCRIPT_TEMPLATE
       git reset --hard $CIRRUS_CHANGE_IN_REPO
     fi
 
+trigger_gcp_pre_prod_deployer_release_task: &TRIGGER_GCP_PRE_PROD_DEPLOYER_RELEASE_TASK
+  # only_if: $TRIGGER == "PUSH_STAGING_DEPLOYER" || $BRANCH == "master"
+  ec2_instance:
+    <<: *VM_TEMPLATE
+  login_to_gcr_script:
+    - export DOCKER_GCLOUD_PASSWORD=$(echo ${DOCKER_GCLOUD_SA_KEY} | base64 -d)
+    - docker login -u _json_key -p "$DOCKER_GCLOUD_PASSWORD" https://${GCLOUD_REGISTRY}
+  install_helm_script:
+    - curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+    - chmod 700 get_helm.sh
+    - ./get_helm.sh
+  helm_dependency_build_script:
+    - cd charts/sonarqube-dce
+    - helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+    - helm repo add bitnami https://charts.bitnami.com/bitnami
+    - helm repo add bitnami-pre2022 https://raw.githubusercontent.com/bitnami/charts/pre-2022/bitnami
+    - helm dependency build
```

Comment:
Same here could be cached, lets consider for hardening ?

Reply:
Yup, SONAR-21764